examples/toolchains/nodejs/WORKSPACE

```diff
@@ -35,11 +29,17 @@ nixpkgs_git_repository(
 )
 
 load("@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl", "nixpkgs_nodejs_configure")
-
 nixpkgs_nodejs_configure(
+  name = "nixpkgs-nodejs_linux_amd64",
   repository = "@nixpkgs",
 )
 
-load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "yarn_install")
+load("@build_bazel_rules_nodejs//:index.bzl", "npm_install")
+npm_install(
+    name = "npm",
+    exports_directories_only = True,
+    node_repository = "nixpkgs-nodejs",
```

Comment:
The example is designed to test both a with-nix and a without-nix configuration. In the latter case this should use the default bindist `node_repository`, so should be set to `None`.

Reply:
Oh, I see it's [currently not tested without Nix](https://github.com/tweag/rules_nixpkgs/blob/dfdc4c7c786822c57a11614f7a066fa042b3051d/.github/workflows/workflow.yaml#L93). Sorry, misread that before. In that case we don't need to worry about this.